source/reference/minio-mc/mc-event-add.rst

```diff
@@ -96,38 +95,30 @@ Parameters
 
    Specify multiple value using a comma ``,`` delimiter.
 
-   Defaults to ``put,delete,get``.
+   If note specified, defaults to ``put,delete,get``.
 
-   See :ref:`mc-event-supported-events` for a detailed list of S3 events
-   associated to each of the supported values.
+   See :ref:`mc-event-supported-events` for a detailed list of S3 events associated to each of the supported values.
 
 .. mc-cmd:: ignore-existing, p
+   :optional:
 
-
-   *Optional* Directs MinIO to ignore applying the specified event
-   triggers if an existing matching trigger exists.
+   Directs MinIO to ignore applying the specified event triggers if an existing matching trigger exists.
```

Comment:
thinking... is `applying` really doing a lot here?
```suggestion
   Directs MinIO to ignore the specified event triggers if an existing matching trigger exists.
```

I'm also wondering if `if an existing matching trigger exists` could be something like `if a matching trigger already exists`.

Reply:
Sure, added `already` and simplified the wording.